scripts/exregional_point_source.sh

```diff
@@ -148,11 +152,11 @@ mkdir_vrfy -p "${DATA}/PT"
 #
 #-----------------------------------------------------------------------
 #
-PREP_STEP
-eval ${RUN_CMD_AQM} ${EXECdir}/decomp-ptemis-mpi ${REDIRECT_OUT_ERR} || \
-print_err_msg_exit "\
-Call to execute PT_SOURCE for Online-CMAQ failed."
-POST_STEP
+#PREP_STEP
+#eval ${RUN_CMD_AQM} ${EXECdir}/decomp-ptemis-mpi ${REDIRECT_OUT_ERR} || \
+#print_err_msg_exit "\
+#Call to execute PT_SOURCE for Online-CMAQ failed."
+#POST_STEP
 
```

Comment:
@bbakernoaa, why do you comment out these commands?

Reply:
whoops this is suppose to be part of https://github.com/ufs-community/ufs-srweather-app/pull/524

I'll remove this from this PR and keep it there.

Reply:
yes. I think Lines 113-116 should be removed too.